Geometry_Engine/Query/GeometricalRepresentation/Point.cs

```diff
@@ -0,0 +1,49 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Geometry;
+using BH.oM.Reflection.Attributes;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+
+namespace BH.Engine.Geometry
+{
+    public static partial class Query
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        [Description("Returns the geometrical representation of the point, which is a Sphere.")] // in the future, we might want an option to choose between sphere / box.
+        public static GeometricalRepresentation Representation(this Point point, ElementRepresentationOptions options = null)
```

Comment:
Would be extremely careful with methods like this, as discussed in call with @rolyhudson .

Having a geometrical representation that is different from the defining geometry for something like a Nod/Bar/Panel/Duct etc object is to me very different from us starting to change them for core geometry. I do not think we really ever ought to add any geometrical representation methods for pure geometry. If we want them to render differently in the UIs, I would see that as a pure rendering/UI issue, and not something that should be converting the geometry into something else in the engine.

Basically, a point needs to keep on being a point and a curve needs to keep on being a curve. If we need them to be meshes/breps/spheres/something else for some situation I think that _needs_ to be handled on toolkit level and not be made central like this.